### PR TITLE
Fix the gateway calculator method.

### PR DESCRIPTION
**Pull request by dansajner**
We were making an assumption that the subnet we a /24. 👊

### CODE REVIEW

File: plugins/ip.rb [hunk 1]
```diff
@@ -32,9 +18,8 @@ def self.pre_validate options
 
     # if we don't have a gateway address we'll compute one
     if ! options[:gateway]
```
Comment (skpy) [inline plugins/ip.rb]: You technically don' t even need this conditional since you've removed the argument to set this.

**Comment by skpy** [comment]
I would prefer to allow manually overriding any computed gateway, since users of this may have routers on addresses other than the computed value.  But this looks legit, otherwise.

**Comment by dansajner** [comment]
@skpy that option is still in place at https://github.com/covermymeds/mkvm/blob/master/lib/kickstart.rb#L21

This just remove the gateway_octet option which was used in the errant calculation.

**Comment by adamcstephens** [comment]
🍕

**Pull request merged**